tests/test_cameras/test_cameras.py

```diff
@@ -163,3 +163,59 @@ def test_perspective():
     cam2.to_screen_()
     cam2.to_ndc_()
     check_camera_close(cam, cam2)
+
+
+def test_perspective_projection():
```

Comment:
Add an perspective projection unit test but unable to pass, please help take a look @JasonBoy1